app/routes.py

```diff
@@ -1,2 +1,328 @@
-from flask import Blueprint
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from datetime import datetime
+from flask import Blueprint, request, jsonify
+from flask.helpers import make_response
+import os
+import requests
 
+# set up blueprints
+# wave 1 (customers & videos)
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+# wave 2 (rentals)
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+# helper function to check that customer/video ids are integers
+def is_int(value):
```

Comment:
👍 heck yeah! This is fine just where it is; however, if you want to really be all about that "modular" life, any helper functionsthatt aren't specific to a model, we can put them in their separate file and import them in.